pronto_mia/lib/core/services/configuration_service.dart

```diff
@@ -1,9 +1,15 @@
 import 'package:global_configuration/global_configuration.dart';
 import 'package:flutter/foundation.dart';
 
+/// A service, that is responsible for accessing the configuration.
+///
+/// Acts as a wrapper for [GlobalConfiguration].
 class ConfigurationService {
   GlobalConfiguration _configuration = GlobalConfiguration();
 
+  /// Loads configuration files according to the current build mode
+  ///
+  /// Takes a [GlobalConfiguration] as an input, to overwrite the current one.

```

Comment:
I removed the argument because it is actually not used anywhere. I initially included this to be able to test the service. Because GlobalConfiguration is not mockable and therefore ConfigService not testable at the moment it is not needed anymore.